packages/rich-text/src/helpers/editor.ts

```diff
@@ -42,6 +42,15 @@ export function getNodeEntryFromSelection(
   return [];
 }
 
+export function isNodeTypeSelected(
+  editor: SPEditor,
+  nodeType: BLOCKS | INLINES
+): boolean {
+  if (!editor) return false;
+  const [table] = getNodeEntryFromSelection(editor, nodeType);
```

Comment:
Not sure why the variable name is `table` here? A bit confusing, maybe `node` or `entry`?

Reply:
Ha, good catch 👍